src/styles/nav.scss

```diff
@@ -0,0 +1,39 @@
+.nav {
+  cursor: pointer;
```

Comment:
The 'cursor: pointer' property should be applied to the '&__link' block instead of the '.nav' block, as it is more appropriate for individual clickable items.